portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,145 @@
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <nav id="outter-nav">
+      <a id="home-logo" href="https://8080-dot-12459592-dot-devshell.appspot.com/">
+        ZG</a>
+      <div id="inner-nav">
+        <div><a class="nav-lnk" href="#about">About</a></div>
+        <div><a class="nav-lnk" href="#experience">Experience</a></div>
+        <div><a class="nav-lnk" href="#projects">Projects</a></div>
+        <div><a class="nav-lnk" href="#contact">Contact</a></div>
+        <div><a class="nav-lnk" href="zghera-resume.pdf">Resume</a></div>
+      </div>
+    </nav>
+    <div>
+      <main>
+        <section id="header" class="content">
+          <p>Hi, my name is</p>
+          <h1>Zach Ghera</h1>
+        </section>
+    
+        <section id="about" class="content">
+          <h2>About Me</h2>
+          <img src="images/zghera-wide.jpg" alt="Me">
+          <p>Hello there! I'm Zach, a rising junior at Purdue University studying
+            Computer Engineering.</p>
+          <p>I have a passion for technology and solving complex problems through 
+            leading and collaborating with others. More specifically, I have been
+            actively pursuing and leading various projects focused on robotic systems
+            outside of classes.
+          </p>
+        </section>
+ 
+        <section id="experience" class="content">
+          <h2>Work Experience</h2>
+          <section class="content">
+            <h3>STEP Intern @ Google</h3>
+            <h4>May - Aug 2020</h4>
+            <p>...</p>
+            <img src="images/trips.png" alt="Google Trips Logo">
+          </section>
+          <section class="content">
+            <h3>EP Intern @ Google</h3>
+            <h4>May - Aug 2019</h4>
+            <p>The purpose of my project was to optimize the core database 
+            implementation of Sawmill Logs, an exabyte scale data lake that supports 
+            internal analytics. Some of my accomplishments include: 
+            </p>
+            <ul>
+              <li>Developed and tested a reusable API for Zstd dictionary compression 
+                in C++ and applied it to the primary storage format for Google logs database. 
+                This minimized compression overhead on small data by training Zstd dictionaries 
+                for certain log types with structured data. This was implemented for standard 
+                compression and compression streams
+              </li>
+              <li>Created performance benchmarking tool to measure the benefits of the new design.
+                Analyzed and visualized the performance results using Python and SQL
+              </li>
+              <li>Improved the compression ratio by 10-15% for certain log storage types with minimal
+                performance reduction
```

Comment:
Nit: Add periods to the end of each sentence. If you prefer not having periods, that is OK as well. Just wanted to point it out in case it wasn't intentional (the bullet points below have periods, FWIW).

Reply:
Yes. I corrected them in PR #2 but did not commit the same changes in this branch. Done.